packages/hardhat/test/setup.ts

```diff
@@ -244,8 +245,8 @@ export const addSqueethLiquidity = async(
       token0,
       token1,
       fee: feeTier,
-      tickLower: -887220,// int24 min tick used when selecting full range
-      tickUpper: 887220,// int24 max tick used when selecting full range
+      tickLower: is3000Fee ? -887220: -887200,// int24 min tick used when selecting full range
+      tickUpper: is3000Fee ? 887220: 887200,// int24 max tick used when selecting full range
```

Comment:
does this only work for 3% and 1% pool ticks? what if we do different pools down the line?

Reply:
yeah this was hard for me to figure out why I was getting mint failures but the tick has to be divisible by the tick spacing

so implication for front end is that each mint to a different pool requires different tick upper / tick lower limits for full range liquidity